prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/PR/PR-receiver-transform.yml

```diff
@@ -0,0 +1,14 @@
+hl7Class: ca.uhn.hl7v2.model.v251.message.ORU_R01
+
+extends: classpath:/metadata/hl7_mapping/ORU_R01/ORU_R01-base.yml
+
+constants:
+  hl7OBXField: '/PATIENT_RESULT/ORDER_OBSERVATION/OBSERVATION/OBX'
+
+elements:
+
+  - name: pr-patient-race-coding-system
```

Comment:
We should look into moving this transform into a an enrichment schema so we can re-use it for everybody. Instead of adding it to every STLT.